packages/now-cli/package.json

```diff
@@ -101,8 +101,8 @@
     "@types/which": "1.3.2",
     "@types/write-json-file": "2.2.1",
     "@vercel/frameworks": "0.0.18-canary.5",
+    "@vercel/ncc": "0.23.0",
```

Comment:
We had to pin ncc in CLI to this older version for a specific reason, but I can't recall exactly what that reason was at the moment.